GenericsHomework.Tests/GenericsHomework.Tests.csproj

```
@@ -0,0 +1,33 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <IsPackable>false</IsPackable>
+	<Nullable>enable</Nullable>
+	<LangVersion>12.0</LangVersion>
+	<TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+	<CodeAnalysisTreatWarningsAsErrors>true</CodeAnalysisTreatWarningsAsErrors>
+	<EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
+	<ImplicitUsings>enable</ImplicitUsings>
+	<EnableNETAnalyzers>true</EnableNETAnalyzers>
+	<IsTestProject>true</IsTestProject>
+  </PropertyGroup>
```

```suggestion
    <IsPackable>false</IsPackable>
	<IsTestProject>true</IsTestProject>
  </PropertyGroup>
```
Since these are in your .props file, you shouldn't need them here